ignite/metrics/mean_absolute_error.py

```diff
@@ -15,22 +15,22 @@ class MeanAbsoluteError(Metric):
 
     where :math:`y_{i}` is the prediction tensor and :math:`x_{i}` is ground true tensor.
 
-    - ``update`` must receive output of the form ``(y_pred, y)`` or ``{'y_pred': y_pred, 'y': y}``.
+    - ``update`` must receive output of the form ``(y_pred, y)``.
 
     Args:
         output_transform: a callable that is used to transform the
             :class:`~ignite.engine.engine.Engine`'s ``process_function``'s output into the
             form expected by the metric. This can be useful if, for example, you have a multi-output model and
             you want to compute the metric with respect to one of the outputs.
-            By default, metrics require the output as ``(y_pred, y)`` or ``{'y_pred': y_pred, 'y': y}``.
+            By default, metrics require the output as ``(y_pred, y)``.
```

Comment:
It is only `update` function that can't handle dicts, but engine's output is routed at first into `Metric.iteration_completed` which can handle dicts. So, this update is not correct.